### PR TITLE
Public

### PR DESCRIPTION
**Pull request by Lexxus007**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+					Moves:<span id="movesCountField">0</span>
+				</div>
+				<div>
+					Time: <span id="timeCountField">00:00</span>
+				</div>
+			</div>
+			
+			<div class="screenField">
+				Top 5 by moves
+				<br/>
```
Comment (Dwarfess) [inline index.html]: It would be better to wrap "Top 5 by moves" by tag with class and set margin/padding than set two "br"

File: js/main.js [hunk 2]
```diff
@@ -0,0 +1,504 @@
+let maxBoardSize = 520;													//максимальний розмір дошки у px, висота або ширина
```
Comment (Dwarfess) [inline js/main.js]: Bad practice to keep all of functionality in one main file. You should separate logic to different files if it's possible

File: js/main.js [hunk 3]
```diff
@@ -0,0 +1,504 @@
...
+let tileSize;															//розмір плитки у пікселях
+let tileMoveTime = 90;													//час на пересування плитки
```
Comment (Dwarfess) [inline js/main.js]: As I see, this variable doesn't change during whole the code, that's why it must have been const

Comment (Dwarfess) [reply js/main.js]: And the same about the rest of the variables

File: js/main.js [hunk 4]
```diff
@@ -0,0 +1,504 @@
...
+	tiles.forEach(tile => {
+		if (tile.tileElement) {		
+			board.append(tile.tileElement);
+		}
+	});	
+}
+
+function addMoveTilesEvents(){
+	board.addEventListener("click", function(e){
+		if (!puzzleIsSolved && e.target.connectedTile){
```
Comment (Dwarfess) [inline js/main.js]: Looks like you have duplicated code 100-105 and 108-113. You should move it to saparate method if it's possible

File: js/main.js [hunk 5]
```diff
@@ -0,0 +1,504 @@
...
+let tiles;												//масив всіх плиток
+let puzzleIsSolved;											//перевірка чи розв'язана головоломка
+
+addMoveTilesEvents();
+restoreCurrentTilesPosition();
+makeStartBoardAndOptions(savedPositionData);
+updateRecordsDataFromStorage();
+updateRecordsFields();
+
+totalRowsField.addEventListener("change", function(){
```
Comment (Dwarfess) [inline js/main.js]: Move all of listeners to separate method this you call only once during opening your app. Like render method

File: index.html (hunk 1 above)
Comment (Lexxus007) [reply index.html]: Done

**Pull request merged**